src/modules/newsletter-subscription/newsletter-subscripion.controller.ts

```diff
@@ -4,6 +4,7 @@ import { CreateNewsletterSubscriptionDto } from './dto/create-newsletter-subscri
 import { skipAuth } from '../../helpers/skipAuth';
 import { ApiOperation, ApiParam, ApiResponse, ApiTags } from '@nestjs/swagger';
 import { SuperAdminGuard } from '../../guards/super-admin.guard';
+import { NewsletterSubscriptionResponseDto } from './dto/newsletter-subscription.response.dto';
```

Comment:
Please why are we reviewing NewsLetter code in adding roles to users

Reply:
I created the branch from the branch I used for implementing newsletter subscription